README.md

```diff
@@ -1,2 +1,2 @@
 # rbac-api-utils
-A library for multi cluster rbac api
+A library for fine-grained multi cluster rbac api
```

Comment:
Are there any usage scenarios that could be provided beyond just letting developers know they need to run `make test`.  So something like:

This is an example spec of a `Role` and `RoleBinding` object that can be used to provide fine grained access to user `fred` for managed cluster `dev-cluster` and namespace `app-for-fred`.

```
role and role binding here
```

Reply:
@gparvin  I added doc for the function inline with details of the input and out params.
I also added an example scenario with role, role bindings and results for sample invocations of the API
Could you pls take a look again

Reply:
Thanks!